### ISSUE TITLE
Accessibility Testing - Level 1

### ISSUE THREAD

**Issue by Sabine-Justilien** [Call-for-Code-for-Racial-Justice/Open-Sentencing#53]
### Background on the problem the feature will solve/improved user experience
Software, whether it is web enabled or not, has the ability to function and behave in a way to help those who have certain disabilities. For example, by incorporating certain bits of code, a blind user can navigate an application and hear a screen reader recite every position their keyboard focus lands. Incorporating other bits of code can help a person who has very low vision to see the screen content; or a person who cannot use a mouse; or a person who is deaf. Even a slide deck used in a presentation can be made accessible; for example, making sure the text colors have a sufficient contrast with the background color, or adding descriptive text into the properties for each image used.

IBM's Equal Access Toolkit provides three levels of achievement for accessibility requirements:

Level 1:  Essential requirements with high user impact, normally with the least investment. These cover safety, color contrast, text alternatives, navigation, operation, and errors.
Level 2:  Adds to Level 1 the next-most important requirements that enable more users to fully use a product. These include pointer interaction, use of color, text style, auto-playing content, and responsive design.
Level 3:  The full set of IBM accessibility requirements that includes the Web Content Accessibility Guidelines (WCAG), US Revised 508, and European EN 301 549 requirements.

WCAG 2.1: Web Content Accessibility Guidelines 2.1  extends WCAG 2.0 and was published as a W3C Recommendation on 05 June 2018. Many regulations reference WCAG 2.1 Level A and AA Success Criteria.

US 508: US Revised 508 Standards provide requirements for information and communication technology (ICT) in the federal sector. 

EN 301 549: Accessibility requirements for ICT products and services, V3.1.1 sets requirements to meet European accessibility regulations. 

Note: The Equal Access Toolkit provides guidance only on WCAG requirements. It divides tasks and considerations by role involved in each phase into these three levels of progression for web applications.

### Describe the solution you'd like
At minimum, complete Level 1 for accessibility testing for Legit Info

### Tasks 
Step 1: Run verification test for Legit Info deployment (you can run a test on your local machine)
You can verify with automated tools, manual inspection, and with a screen reader. It is recommended that you do all three in the given order. It is counterproductive to continue to manual testing until the automated tool produces a report free of Violations or issues that Need Review. Once the automated test returns a clean result, move on to manual tests.

Step 2: Review violations and needs review 
The IBM accessibility requirements and automated tools are numbered and aligned to the international standards, so it makes gathering and recording the results simple. Once you have a list of items that need attention, create a plan of action to resolve issues. Be sure to capture what elements are design, UX/UI, or developer focused activities

Step 3: Resolve violations and needs review items needed to achieve Level 1 accessibility for Legit Info
You can review Level 1 requirements by setting the filter to Level 1 on the lefthand side of the screen. Please ensure all of the boxes for Standards and Technology are checked

### Acceptance Criteria 
See Level 1 accessibility testing requirements

**Comment by Sabine-Justilien** [comment]
@sforsyth089 This work is going to be done in January-March 2021 by a corporate cohort

**Comment by demilolu** [comment]
not stale!
